src/ast/passes/codegen_llvm.cpp

```diff
@@ -3513,18 +3513,23 @@ void CodegenLLVM::createMapDefinition(const std::string &name,
   var->setDSOLocal(true);
   var->addDebugInfo(debuginfo);
 }
+bool CodegenLLVM::needs_percpu_map(const SizedType &val_type)
```

Comment:
Nit: we tend to add empty lines between functions:
```suggestion

bool CodegenLLVM::needs_percpu_map(const SizedType &val_type)
```
(I'm surprised this is not enforced by clang-format)